docs/index.md

```diff
@@ -29,7 +29,7 @@ SLE Micro for Rancher is a containerized and "stripped to the bones" OS layer. A
 It's sole purpose is to run Kubernetes (k3s or RKE2), with everything controlled through Rancher Manager.
 
 Elemental Teal is built in the [openSUSE Build Service](https://build.opensuse.org/package/show/isv:Rancher:Elemental:Stable:Teal53/node-image)
-and available through the [openSUSE Registry](https://registry.opensuse.org/cgi-bin/cooverview?srch_term=project%3D%5Eisv%3ARancher%3AElemental%3AStable+container%3D.*).
+and available through the [SUSE Registry](https://registry.suse.com).
```

Comment:
I am doubtful about this one. If you navigate to `registry.suse.com` from your browser it is not possible to search and find Elemental in there in any way (at least I couldn't). So I am not we should add this link here, probably we could add something like:

```
... and available through the SUSE registry ('registry.suse.com').
```
without an http url.

Reply:
Aren't there CLI tools available to browse a registry ? These should work with registry.suse.com 🤔

Reply:
Why anyway drop the openSUSE registry link? I would add the suse.com but leave the opensuse one too